roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+docker_yum_versionlock_file: /etc/yum/pluginconf.d/versionlock.list
+
+docker_package_names:
```

Comment:
Should we also lock `container-selinux`? Or is that pinned as reqs of these packages?

Reply:
That's what I did at the beginning, but I've removed it because there are already dependencies in docker-ce & docker-engine RPMs, so no need for that, yum versionlock will prevent from upgrading if dependencies would not be satisfied.